examples/puppetmaster/smoketest_e2e.sh

```diff
@@ -79,16 +99,15 @@ convergeNode() {
     password $api_key
   " > $identity_file
 
-  docker run --rm \
+  docker run --rm -t \
     --net $NETNAME \
     -e 'FACTER_CONJUR_SMOKE_TEST=true' \
     -v $config_file:/etc/conjur.conf:ro \
     -v $identity_file:/etc/conjur.identity:ro \
     -v $PWD:/src:ro -w /src \
-    puppet/puppet-agent-ubuntu
+    puppet/puppet-agent-ubuntu:5.5.1
```

Comment:
will this fail if the identity isn't set up correctly? is there any significance to the node version being different than the puppetmaster version?

Reply:
> will this fail if the identity isn't set up correctly?

Node wouldn't be able to fetch the secret afaik without this.

> is there any significance to the node version being different than the puppetmaster version?

I think major versions must equal so we should be fine - this combo works and they are the latest `5.x.y` versions available on dockerhub for their specific image.